services/FileCreate.proto

```diff
@@ -33,7 +33,7 @@ After the file is created, the FileID for it can be found in the receipt, or rec
 
  The file contains the specified contents (possibly empty). The file will automatically disappear at the expirationTime, unless its expiration is extended by another transaction before that time. If the file is deleted, then its contents will become empty and it will be marked as deleted until it expires, and then it will cease to exist.
 
- The keys field is a list of keys. All the keys on the list must sign to create or modify a file, but only one of them needs to sign in order to delete the file.  Each of those "keys" may itself be threshold key containing other keys (including other threshold keys). In other words, the behavior is an AND for create/modify, OR for delete. This is useful for acting as a revocation server. If it is desired to have the behavior be AND for all 3 operations (or OR for all 3), then the list should have only a single Key, which is a threshold key, with N=1 for OR, N=M for AND.
+ The keys field is a list of keys. All keys within the top level key list must sign to create or modify a file, but only one of them is required to sign from the top level key list in order to delete the file.  Each of those "keys" may itself be threshold key containing other keys (including other threshold keys). In other words, the behavior is an AND for create/modify, OR for delete. This is useful for acting as a revocation server. If it is desired to have the behavior be AND for all 3 operations (or OR for all 3), then the list should have only a single Key, which is a threshold key, with N=1 for OR, N=M for AND.
```

Comment:
Suggested Update: "The keys field is a list of keys. All keys within the top level KeyList must sign (M-of-M) to create or modify a file, but only one keys (1-of-M) is required to sign from the top level KeyList in order to delete the file.  Each keys may contains additional KeyList or ThresholdKey. If desired to have the behavior be "AND" (M-of-M) or OR (1-of-M) for all 3 operations then keys should have only a single ThresholdKey containing (N=M) for "AND" or (N=1) for "OR"."